extensions/typescript-language-features/src/utils/plugins.ts

```diff
@@ -28,7 +28,7 @@ namespace TypeScriptServerPlugin {
 export class PluginManager extends Disposable {
 	private readonly _pluginConfigurations = new Map<string, {}>();
 
-	private _plugins: Map<string, ReadonlyArray<TypeScriptServerPlugin>> | undefined;
+	private _plugins?: Map<string, ReadonlyArray<TypeScriptServerPlugin>>;
```

Comment:
Any advantages to `a?: T` instead of `a: T | undefined` in this context other than fewer characters?

Reply:
Not really here. It's more correct in this case because we never initialize the value in the constructor, but our compiler settings don't make TS enforce this